# Constant movement message
Header header
int8[8] states
int8[7] speeds
bool query
bool quit